### PR TITLE
feat(pie-docs): DSW-2475 toast code page

### PR DESCRIPTION
**Pull request by thejfreitas**
Created the toast code in the documentation website

### Screenshots
Screenshot 2024-11-15 at 9 58 08 AM
Screenshot 2024-11-15 at 9 58 01 AM

## Author Checklist (complete before requesting a review)
- [x] I have performed a self-review of my code
- [x] I have added thorough tests where applicable (unit / component / visual)
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
- [x] I have reviewed visual test updates properly before approving
- [x] If changes will affect consumers of the package, I have created a changeset entry.

### Reviewer 1 @raoufswe 
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
- [x] If there are visual test updates, I have reviewed them

### Reviewer 2 @siggerzz 
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
- [x] If there are visual test updates, I have reviewed them

### CODE REVIEW

File: apps/pie-docs/component-statuses.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (siggerzz) [inline apps/pie-docs/component-statuses.json:1]: This file can be removed 🎉.

A PR was recently introduced to change how component statuses are fetched https://github.com/justeattakeaway/pie/pull/2034 :)

File: apps/pie-storybook/component-statuses.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (siggerzz) [inline apps/pie-storybook/component-statuses.json:1]: ditto

File: component-statuses.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (siggerzz) [inline component-statuses.json:1]: ditto

File: apps/pie-docs/component-statuses.json (hunk 1 above)
Comment (raoufswe) [reply apps/pie-docs/component-statuses.json:1]: deleted

**Review by raoufswe — DISMISSED** [review]

File: apps/pie-docs/src/components/toast/code/props.json [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+                "item": ["\"\""]
+            }
+        ],
+        [
+            "isMultiline",
+            {
+                "type": "code",
+                "item": ["true", "false"]
+            },
+            "It allows the message content being displayed as multiline limited to three rows.",
```
Comment (siggerzz) [inline apps/pie-docs/src/components/toast/code/props.json:70]:
```suggestion
            "Allows the message content to be displayed as multiline, limited to three rows.",
```

File: apps/pie-docs/src/components/toast/code/props.json [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+                "item": ["undefined"]
+            }
+        ],
+        [
+            "duration",
+            {
+                "type": "code",
+                "item": ["number", "null"]
+            },
+            "Sets the duration of the toast in milliseconds before it auto-dismisses. If the value is `null`, auto-dismiss is disabled. If the value is not provided it auto-dismisses after 5 seconds (5000 milliseconds).",
```
Comment (fernandofranca) [inline apps/pie-docs/src/components/toast/code/props.json:94]:
```suggestion
            "Sets the duration of the toast in milliseconds before it auto-dismisses. If the value is `null`, auto-dismiss is disabled. If the value is not provided it auto-dismisses after 5 seconds.",
```

File: apps/pie-docs/src/components/toast/code/props.json (hunk 4 above)
Comment (raoufswe) [reply apps/pie-docs/src/components/toast/code/props.json:70]: updared

File: apps/pie-docs/src/components/toast/code/props.json (hunk 5 above)
Comment (raoufswe) [reply apps/pie-docs/src/components/toast/code/props.json:94]: updated

**Review by raoufswe — APPROVED** [review]

**Pull request merged**